snappiershot/snapshot/metadata.py

```diff
@@ -29,24 +66,36 @@ def __init__(
         self._validate()
 
     def as_dict(self) -> Dict:
-        """ Returns a JSON-serializable dictionary of metadata. """
+        """Returns a JSON-serializable dictionary of metadata."""
         dct = vars(self).copy()
         dct["arguments"] = dct.pop("caller_info").args
         return dct
 
     def matches(self, metadata_dict: Dict) -> bool:
-        """ Check if the "metadata" section of a snapshot file sufficiently matches
-        this metadata object.
+        """Check if the "metadata" section of a snapshot file sufficiently matches
+        the metadata object coming from the test method inputs.
```

Comment:
Suggest to include caveat, "Note, only test metadata is checked for matching snapshot inputs. Snapshot file metadata section may have extra data not in test method inputs and this check will return True"

Reply:
fixed [1fc52b3](https://github.com/MORSECorp/snappiershot/pull/96/commits/1fc52b3cc6a8fc8a141b51ab311dbd3f2fe6b691)